src/app/profile/components/profile/profile.component.ts

```diff
@@ -57,4 +57,7 @@ export class ProfileComponent implements OnInit, OnDestroy {
       .then(() => this.router.navigate(['/dashboard']))
       .catch();
   }
+  openSignal(): void{
+    window.open('https://signal.group/#CjQKIM3ulwlm8OT0_dlVj3GRuZNypumHPahTkJxBCQHtoJSmEhDBzZwFxv4lARp7rG-2vF_p','_blank');
```

Comment:
You could put that link into a property in the environment.ts file(s). 
Would make it easier in case we need to change the link later down the road..